releasenotes/notes/add_snmp_auto_conf-b01f24781429707c.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Move SNMP ``auto_conf.yaml`` from integrations-core to agent repo to have it
+    in the same repository as ``snmp_listener``.
```

Comment:
you should remove this release note since it has no impact on end users of the Agent